core/src/com/unciv/logic/city/CityReligion.kt

```diff
@@ -19,25 +37,69 @@ class CityInfoReligionManager: Counter<String>() {
     }
     
     fun getNumberOfFollowers(): Counter<String> {
-        val totalInfluence = values.sum()
+        val totalInfluence = pressures.values.sum()
         val population = cityInfo.population.population
         if (totalInfluence > 100 * population) {
```

Comment:
Sounds really strange. And doesn't it make getNumberOfFollowers not adding up to cityInfo.population.population by a potentially wide margin? If that's significant at all? The other loop also does not guarantee it adds up, though it will come close...

Reply:
This function was already here before I started working on religion, I actually have no idea what it does and just changed the code based on my change of pressure from an inherited property to a member property. I'll look at it later when I implement cities having multiple religions competing inside them & followers